kubernetes/annotations_and_labels.md

```diff
@@ -51,16 +47,38 @@ This page defines common annotations and labels we set in Kubernetes objects.
   value may be equal to that of `app.kubernetes.io/version` but it has a
   different purpose and since there could be multiple operators reconciling one
   object there could be multiple per-operator labels on one object.
-- `app.giantswarm.io/branch` - (informational) branch from which this
-  instance of the app/operator was built from.
+- `helm.sh/chart` - value should contain a chart name and version, see
+  [Helm chart labels best practices][helm-labels]
+- `app.kubernetes.io/name` - the name of the application; should be applied to
+  every Kubernetes resource associated with that application, i.e. all objects
+  in a chart that installs it.
+- `app.kubernetes.io/instance` - value should be set to `{{ .Release.Name }}`
+  and is meant for differentiating between instances of the same application.
+  This together with `app.kubernetes.io/name` should be used as replicas
+  selector, e.g.
+  `app.kubernetes.io/name=kvm-operator,app.kubernetes.io/instance=kvm-operator-1.0.0`.
+- `app.kubernetes.io/version` and other common labels (see
+  [here][helm-labels] and [here][k8s-common-labels]) - are informational,
+  applied to all objects in a chart that installs an app or operator and so
+  they indicate the source and instance of that app/operator; value of
+  `.../version` should be the app/operator version as defined in `project.go`
```

Comment:
This `.../version` shortcut is confusing IMO.

Reply:
Fair enough, I'll get rid of them.

Reply:
Done